tests/integration/helpers.py

```diff
@@ -0,0 +1,22 @@
+from charms.mongodb.v1.helpers import MONGO_SHELL
+from pytest_operator.plugin import OpsTest
+import ops
+
+MONGOS_URI = (
+    "mongodb://%2Fvar%2Fsnap%2Fcharmed-mongodb%2Fcommon%2Fvar%2Fmongodb-27018.sock"
+)
+MONGOS_APP_NAME = "mongos"
+
+
+async def mongos_command(ops_test: OpsTest) -> str:
```

Comment:
do you plan to make it a function to run any command or it is a function just to ping mongos? 

Not sure because the name and actions are different, maybe it is a good idea to updated the name or add comment :)

Reply:
I can update name!